topic-offset-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/topicoffset/kafka/KafkaTestConfiguration.java

```diff
@@ -43,13 +51,18 @@
 import static java.util.stream.Collectors.toMap;
 import static org.apache.kafka.clients.admin.AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG;
 
+/**
+ * Configuration providing the test beans to be injected into test classes that
+ * require access to various objects to support access to Kafka/Akka
+ */
 @Configuration
 @ConditionalOnProperty(
         value = "akka.discovery.config.services.kafka.from-config",
         havingValue = "false"
 )
 public class KafkaTestConfiguration {
     private static final Logger LOGGER = LoggerFactory.getLogger(KafkaTestConfiguration.class);
+    private static final ObjectMapper MAPPER = new ObjectMapper();
 
     private final List<NewTopic> topics = List.of(
             new NewTopic("masked-resource", 3, (short) 1),
```

Comment:
Standardise number of test topics. some classes have 1 topic, some 3

Reply:
Standardised this value to 1 across all services.